{"name": "测试数据：【 测试test，123456，False，1003，用户名不存在】", "status": "broken", "statusDetails": {"message": "TypeError: login_user() takes 0 positional arguments but 2 were given", "trace": "self = <testcases.api_test.test_03_login.TestUserLogin object at 0x0000016B22546588>, username = '测试test', password = '123456', except_result = False, except_code = 1003, except_msg = '用户名不存在'\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, except_result, except_code, except_msg\",\n                             api_data[\"test_login_user\"])\n    def test_login_user(self, username, password, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n>       result = login_user(username, password)\nE       TypeError: login_user() takes 0 positional arguments but 2 were given\n\ntestcases\\api_test\\test_03_login.py:29: TypeError"}, "description": "该用例是针对获取用户登录接口的测试", "attachments": [{"name": "log", "source": "551e1d06-8cab-4458-afb6-33af362ec913-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'测试test'"}, {"name": "password", "value": "'123456'"}, {"name": "except_result", "value": "False"}, {"name": "except_code", "value": "1003"}, {"name": "except_msg", "value": "'用户名不存在'"}], "start": 1600161454508, "stop": 1600161454508, "uuid": "3e71b2f9-7bb2-436a-a865-3d789769aba7", "historyId": "1225d996d98de9edea8cde3d1115527f", "testCaseId": "957e450fb6ceaa933c3696671a75c4d9", "fullName": "project.pytestDemo-master.pytestDemo-master.testcases.api_test.test_03_login.TestUserLogin#test_login_user", "labels": [{"name": "feature", "value": "用户登录模块"}, {"name": "severity", "value": "normal"}, {"name": "story", "value": "用例--登录用户"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "project.pytestDemo-master.pytestDemo-master.testcases.api_test"}, {"name": "suite", "value": "test_03_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "DESKTOP-68E5LL7"}, {"name": "thread", "value": "8116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "project.pytestDemo-master.pytestDemo-master.testcases.api_test.test_03_login"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}